packages/ui/src/components/va-date-input/VaDateInput.vue

```diff
@@ -84,7 +88,14 @@ export default defineComponent({
     ...VaInputProps,
     resetOnClose: { type: Boolean, default: true },
     isOpen: { type: Boolean },
+
     format: { type: Function as PropType<(date: VaDatePickerModelValue | undefined) => string> },
+    formatDate: { type: Function as PropType<(date: Date) => string>, default: () => (d: Date) => d.toLocaleDateString() },
```

Comment:
Add some details to documentation on how these are different. (maybe more examples?)